### DISCUSSION TITLE
Is paradise template removed from the list

### DISCUSSION THREAD

**Discussion by Github743** [primefaces/community#1715] [PrimeNG Templates] [answered]
I buyed this layout https://www.primefaces.org/paradise-ng/ now when I am trying to open the link it is showing as page not found.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Paradise is outdated and removed from the sale.

**Reply by Github743** [reply]
OK I already purchased this long back, with angular 17 I am facing some issues whom to reach out.
